src/Utils.php

```diff
@@ -29,6 +29,7 @@ public static function contactFieldTypes() {
     return array(
       'Select' => 'select',
       'Multi-Select' => 'checkboxes',
+      'Number' => 'textfield',
```

Comment:
Can we extract that into a separate PR, as it's not semantically related to adding weights …?

Reply:
Also, this _might_ better be mapped to the [`number` element type](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21Element%21Number.php/class/Number/11.x) of Drupal Form API.